core/commands/commit.py

```diff
@@ -64,6 +64,11 @@ def run_async(self, repo_path=None, include_unstaged=False, amend=False):
         else:
             view.set_syntax_file("Packages/GitSavvy/syntax/make_commit.tmLanguage")
 
+        if savvy_settings.get("vintageous_friendly", False) is True:
```

Comment:
Can you add a comment block above this line, explaining why it is here and what it does?  Add the same for `log_graph.py` and `ui.py`.